static.go

```diff
@@ -237,6 +240,37 @@ const (
 				</form>
 			</div>
 		</section>
+		<section class="row">
+			<div id="restore" class="col-md-6">
+				<h3>Restore</h3>
```

Comment:
Is it confusing for this to be called `Restore` when it's really a delegation for a possible restore?